packages/SketchMorph2-Core.package/M2TemporaryTextMorph.class/instance/defaultHandleOffset.st

```diff
@@ -0,0 +1,3 @@
+defaults
+defaultHandleOffset
+	^ self topCenter - (self defaultHandleExtent * (0.5@1))
```

Comment:
I think putting the handle at the bottom right corner (like we do with the canvas grip morph) would be better, but that's just personal preference. Maybe the customer has an opinion on that?